src/middlewares/__tests__/secretsManager.js

```diff
@@ -52,6 +52,7 @@ describe('🔒 SecretsManager Middleware', () => {
       promise = promise.then(() => {
         return new Promise((resolve, reject) => {
           handler(event, context, (error, response) => {
+            if (error) reject(error)
```

Comment:
Should there be `return` before this `reject`?

Reply:
Yes, I must've deleted it when cleaning up.
Thanks!